lib/launcher.js

```diff
@@ -64,6 +65,8 @@ class Launcher {
         }
         nodesDir.push(path.join(require.main.path, 'node_modules', '@flowforge', 'nr-theme').replace(/\\/g, '/')) // MVP: fixed to loading FF theme
         nodesDir.push(path.join(require.main.path, '..', 'nr-theme').replace(/\\/g, '/')) // MVP: fixed to loading FF theme
+        nodesDir.push(path.join(require.main.path, 'node_modules', '@flowforge', 'nr-project-nodes').replace(/\\/g, '/'))
```

Comment:
How do the project nodes get installed in the launcher?

Reply:
That's how I was running from local file system. I think I need to move to the new project and possibly remove this line. Let me check.

Reply:
I wasn't commenting on that specific line as such... just the point in the code I got to when the question came up in my mind.

Are they installed, like the theme, via this package's `dependencies`? I don't see an update to include them in that file.

Reply:
> Are they installed, like the theme, via this package's dependencies?
In my current environment yeah - but...

> I don't see an update to include them in that file
I "lost" the `package.json` changes due to having to ditch it for merging main! DOH!
Also, I need to discuss with @hardillb as there was an issue with the method used for theme when deployed to cloud.

Reply:
### Docker + k8
Once published to NPM, to deps to ...
* https://github.com/flowforge/docker-compose/blob/main/node-red-container/package.json
* https://github.com/flowforge/helm/blob/main/node-red-container/package.json

### Local-fs
At present, we hard code paths in the source (that this PR does) - but currently need to add dep to 
* https://github.com/flowforge/flowforge-nr-launcher/blob/main/package.json

### Future...
* Need to revisit method of picking up nodes instead of adding hardcoded paths.
* As discussed with @hardillb - aim for solution for v0.9

Reply:
re last point: https://github.com/flowforge/flowforge-nr-launcher/issues/54